mesa/space.py

```diff
@@ -502,7 +503,8 @@ def position_agent(
         self, agent: Agent, x: int | str = "random", y: int | str = "random"
     ) -> None:
         """Position an agent on the grid.
-        This is used when first placing agents! Use 'move_to_empty()'
+        This is used when first placing agents! Setting either x or y to "random"
+        gives the same behavior as 'move_to_empty()'.
         when you want agents to jump to an empty cell.
```

Comment:
You forgot to delete L508, a dangling sentence.

Reply:
Thanks for pointing this out!
[f6d9667](https://github.com/projectmesa/mesa/pull/1457/commits/f6d96677beea8d9b43d36f9ae2326cbf0cc8381e) - The dangling sentence deleted.